### DISCUSSION TITLE
可以把知识库的配置改成动态加载吗

### DISCUSSION THREAD

**Discussion by flizzywine** [wenda-LLM/wenda#291] [Ideas]
由于对知识库运用不熟悉，需要不断调整知识库的配置。每次调整之后，都要重新启动，每次加载模型就要一分钟以上，较为耗时，能把知识库的配置，改成web动态修改吗？

**Comment by l15y** [comment]
有这个接口，可以通过auto调用。等有空可以做到ui里
